test/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -456,23 +456,25 @@ public void InstallCommand_GetLastestPrereleaseVersion()
                 var package2 = new ZipPackage(packageFileName);
                 MachineCache.Default.RemovePackage(package2);
 
-                var server = Util.CreateMockServer(mockServerEndPoint, new [] { package1, package2 });
-                server.Start();
-                
-                // Act
-                var args = "install testPackage1 -Prerelease -Source " + mockServerEndPoint + "nuget";
-                var r1 = CommandRunner.Run(
-                    nugetexe,
-                    workingDirectory,
-                    args,
-                    waitForExit: true);                
-                server.Stop();
-
-                // Assert
-                Assert.Equal(0, r1.Item1);
-
-                // testPackage1 1.2.0-beta1 is installed
-                Assert.True(Directory.Exists(Path.Combine(workingDirectory, "testPackage1.1.2.0-beta1")));
+                using (var server = Util.CreateMockServer(new[] { package1, package2 }))
+                {
+                    server.Start();
+
+                    // Act
+                    var args = "install testPackage1 -Prerelease -Source " + server.Uri + "nuget";
+                    var r1 = CommandRunner.Run(
+                        nugetexe,
+                        workingDirectory,
+                        args,
+                        waitForExit: true);
+                    server.Stop();
```

Comment:
Right because of that I missed it, no need to call server stop anymore it's part of the dispose method